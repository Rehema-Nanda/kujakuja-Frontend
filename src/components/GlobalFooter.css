
.footer{
    width: 100%;
    height: 144px;
    background-color: #2B3841;
    padding: 32px;
    display: flex;
    align-items: center;
}

.footer-details{
    width: 100%;
    height: 64px;
    display: flex;
}

.footer-logo{
    width: 64px;
    height: 64px;
}

.footer-social{
    flex: 1;
    display: flex;
    align-items: center;
}

.social-instagram{
    width: 16px;
    height: 16px;
    margin: 16px;
    background-image: url('../img/icons/social_instagram_off.svg');
    background-size: cover;
    cursor: pointer;
    transition: 0.3s;
    margin-left: auto;
}

.social-instagram:hover{
    background-image: url('../img/icons/social_instagram_on.svg');
}

.social-fb{
    width: 16px;
    height: 16px;
    margin: 16px;
    background-image: url('../img/icons/social_fb_off.svg');
    background-size: cover;
    cursor: pointer;
    transition: 0.3s;
}

.social-fb:hover{
    background-image: url('../img/icons/social_fb_on.svg');
}

.social-twitter{
    width: 18px;
    height: 16px;
    margin: 16px;
    background-image: url('../img/icons/social_twitter_off.svg');
    background-size: cover;
    background-repeat: no-repeat;
    background-size: 100%;
    cursor: pointer;
    transition: 0.3s;
}

.social-twitter:hover{
    background-image: url('../img/icons/social_twitter_on.svg');
}

.social-medium{
    width: 16px;
    height: 16px;
    margin: 16px;
    background-image: url('../img/icons/social_medium_off.svg');
    background-size: cover;
    cursor: pointer;
    transition: 0.3s;
}

.social-medium:hover{
    background-image: url('../img/icons/social_medium_on.svg');
}