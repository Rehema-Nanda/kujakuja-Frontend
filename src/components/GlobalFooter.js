
import React from 'react';

import './GlobalFooter.css';

export default class GlobalFooter extends React.Component {

    constructor(props) {
        super(props);
        this.state = {
        };
    }

    render() {
        return (

            <section className="footer">
                <div className="footer-details">
                    <div className="footer-logo"><img src={require('../img/footer-logo.svg')} alt="idea feed" /></div>
                    <div className="footer-social">
                        <div className="social-instagram" onClick={() => window.open('https://www.instagram.com/kujakujaglobal/', '_blank')}></div>
                        <div className="social-fb" onClick={() => window.open('https://www.facebook.com/KujaKujaGlobal', '_blank')}></div>
                        <div className="social-twitter" onClick={() => window.open('https://twitter.com/kujakujaglobal', '_blank')}></div>
                        <div className="social-medium" onClick={() => window.open('https://medium.com/@KujaKujaGlobal', '_blank')}></div>
                    </div>
                </div>
            </section>

        );
    }
}
