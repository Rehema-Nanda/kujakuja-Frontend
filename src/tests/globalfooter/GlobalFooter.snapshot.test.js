import React from 'react';
import renderer from 'react-test-renderer';
import GlobalFooter from '../../components/GlobalFooter';

it('renders correctly', () => {
    const tree = renderer.create(
        <GlobalFooter/>
    ).toJSON();

    expect(tree).toMatchSnapshot();
});
