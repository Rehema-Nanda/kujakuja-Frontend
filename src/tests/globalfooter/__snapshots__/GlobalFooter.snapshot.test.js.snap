// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="footer"
>
  <div
    className="footer-details"
  >
    <div
      className="footer-logo"
    >
      <img
        alt="idea feed"
        src={
          Object {
            "ReactComponent": Object {
              "$$typeof": Symbol(react.forward_ref),
              "render": [Function],
            },
            "__esModule": true,
            "default": "footer-logo.svg",
          }
        }
      />
    </div>
    <div
      className="footer-social"
    >
      <div
        className="social-instagram"
        onClick={[Function]}
      />
      <div
        className="social-fb"
        onClick={[Function]}
      />
      <div
        className="social-twitter"
        onClick={[Function]}
      />
      <div
        className="social-medium"
        onClick={[Function]}
      />
    </div>
  </div>
</section>
`;
